tools/generator/test/DisambiguateTargetsTests.swift

```diff
@@ -60,11 +71,20 @@ final class DisambiguateTargetsTests: XCTestCase {
             "B": Target.mock(
                 product: .init(type: .staticLibrary, name: "B", path: "")
             ),
+            // The following targets only differ by case
+            "C 1": Target.mock(
+                product: .init(type: .staticLibrary, name: "C", path: "")
+            ),
+            "C 2": Target.mock(
+                product: .init(type: .application, name: "c", path: "")
+            ),
         ]
         let expectedTargetNames: [TargetID: String] = [
             "A 1": "A (Library)",
             "A 2": "A (App)",
             "B": "B",
+            "C 1": "C (Library)",
+            "C 2": "c (App)",
```

Comment:
@brentleyjones I just want to confirm that this is the output you would expect for the `producType` test. If so, I will need to modify the following conditional 
```swift
if labelsByName[normalizedName]!.count == 1 {
```

Reply:
That is the expected output, yes.

Reply:
That condition is fine. We just need to store normalized labels in it here:
```
labelsByName[target.name, default: []].insert(target.label)
```

Reply:
And then the label test will need to use unique (not just case differences) labels.